### DISCUSSION TITLE
Frigate thinks seeing a person 70/80%, but it is shadow...

### DISCUSSION THREAD

**Discussion by Operations8** [blakeblackshear/frigate#7530] [Ask A Question]
I am getting a lot of Frigate event (every couple of minutes) hits on "person" between 70/80%. But all that is happing is moving shadow from leaves of the nearby tree.

Any ideas on how to fix this? 80% person but only leaf shadow is a pretty high wrong number... right?

**Comment by NickM-27** [comment]
False positives are not unexpected with the default model because the default model is a Google demo model trained on the COCO dataset. [If you look at some examples of images in the COCO dataset](https://imgur.com/a/CNImofJ) they do not look like the security camera images at all. This leads to false positives. 

There are a number of filters and other options to reduce false positives covered in the docs. 

Better models, like [what Frigate+ is planning to offer](https://docs.frigate.video/integrations/plus), that are trained on actual security camera examples offer higher accuracy and less issues with false positives like this.

**Comment by Operations8** [comment]
Nick i am trying to create an account for Frigate+ but i am not receiving the confirmation email. Do you know which domain the email will come from? Maybe it will help if i whitelist the frigate domain. (Frigate.com /frigate.video?)

**Reply by NickM-27** [reply]
frigate.video I believe but not 100% sure, been a long while since I did that

**Comment by Operations8** [comment]
> frigate.video I believe but not 100% sure, been a long while since I did that

Frigate+ Plus not yet available, do you have any idea when this will be available?

I am trying to create a zone now to reduce the false positives. But when i create a zone is this to exclude this area or to make this the zone frigate will look at?

**Reply by NickM-27** [reply]
I don't work on Frigate+, I believe it was said it will come out alongside 0.13

Zones make it so you can only have events created for objects that are detected in areas you care about.

**Comment by Boldfor** [comment]
> I am getting a lot of Frigate event (every couple of minutes) hits on "person" between 70/80%. But all that is happing is moving shadow from leaves of the nearby tree.

Here's my example of a person (with 96% (!) confidence):

![image](https://github.com/user-attachments/assets/1b276e37-47ec-4b78-907a-3241e6d5d244)

Here's my config for the model/detection:
```
# Configure the model for the Hailo 8l detector
model:
  width: 300
  height: 300
  input_tensor: nhwc
  input_pixel_format: bgr
  model_type: ssd
  path: /config/model_cache/h8l_cache/ssd_mobilenet_v1.hef
```

In case someone has an idea, let me know.
